scripts/start.sh

```
@@ -0,0 +1,25 @@
+  | docker login --username AWS --password-stdin $(echo $IMAGE_URI | cut -d'/' -f1)
+
+echo "Pulling image: $IMAGE_URI"
+docker pull $IMAGE_URI
+
+echo "Starting container: $CONTAINER_NAME"
+docker run -d \
+  --name $CONTAINER_NAME \
+  -p 8080:8080 \
+  --env-file $ENV_FILE \
+  --restart unless-stopped \
+  $IMAGE_URI
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)